daprdocs/content/en/developing-applications/ides/visual-studio-code/vscode-manual-configuration.md

```diff
@@ -1,64 +1,11 @@
 ---
 type: docs
-title: "Visual Studio Code integrations with Dapr"
-linkTitle: "Visual Studio Code"
-weight: 1000
-description:  "Information on how to develop and run Dapr applications in VS Code"
+title: "Visual Studio Code manual configuration"
+linkTitle: "Visual Studio Code manual configuration"
```

Comment:
```suggestion
linkTitle: "Manual debugging"
```

Reply:
This one I think is best as "Manual debugging" as it's shorter and won't word wrap on narrow screens